test/Aux.hs

```diff
@@ -0,0 +1,123 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE ImportQualifiedPost #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+
+module Aux (runBuiltin) where
+
+import Control.Monad.Error.Class (liftEither)
+import Data.Aeson qualified as Aeson
+import Data.Bifunctor (Bifunctor (first))
+import Data.Coerce (Coercible, coerce)
+import Data.Default.Class (Default (def))
```

Comment:
Do we need the whole type class, or just `def`? If so, we don't need to import the type class as well.